### ISSUE TITLE
Chats in convo list disappear after migration

### ISSUE THREAD

**Issue by cstiens** [zom/Zom-Android-XMPP#241]
Here’s the scenario: All of Jodie’s chats in the convo went away after she migrated. During the migration, the chats in her convo view remained and showed the upgrade message going out from her dukgo account. After the migration was done, they disappeared. Once friends accepted the her new account, the new chat with them showed up in the convo view. 

@n8fr8 Is this your desired workflow for Android? On iOS we show pre-existing chats with ‘pending approval’ next to it in the convo view after migration.

**Comment by n8fr8** [comment]
Yes I think so. I thought we were supposed to archive the chats from the old account? The "pending approval" issue is related to the fact we show that in the 2nd tab, and not the first.

**Comment by cstiens** [comment]
> I thought we were supposed to archive the chats from the old account?

Yes, that's right. On iOS though, we keep them as 'placeholders' with pending approval marked next to the conversation. Then once approved, that chat would take you into the new chat with them. This way, it doesn't make it look like all of your conversations disappear after you migrate. 

I'm marking this as low-priority for now. In the interest of getting a build out, I don't think this is a deal breaker.

**Comment by n8fr8** [comment]
Just to clarify, you essentially create *new* conversations with the same people you had the old conversations open with? The old conversations from the old account with them are archived.

**Comment by cstiens** [comment]
All chats disappear after migration. To begin a new chat after migration you must start a chat from the friends' view or they can begin one with you.
